src/main/java/io/opentracing/contrib/global/concurrent/TracedCallable.java

```diff
@@ -0,0 +1,70 @@
+package io.opentracing.contrib.global.concurrent;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer.SpanBuilder;
+import io.opentracing.contrib.global.GlobalTracer;
+
+import java.util.concurrent.Callable;
+
+import static io.opentracing.contrib.global.concurrent.AddSuppressedSupport.addSuppressedOrLog;
+
+/**
+ * Convenience {@link Callable} wrapper that will execute within a new {@link Span} if an
+ * {@link #withOperationName(String) operationName} is provided.<br>
+ * If no operationName is provided, the call will be executed without starting a new Span.
```

Comment:
Is the use of Tracer.buildSpan(null) (null for operationName) considered accepted behaviour?
I agree with you if it is.